core/amber/src/main/python/core/util/console_message/replace_print.py

```diff
@@ -53,7 +53,7 @@ def wrapped_print(*args, **kwargs):
                 complete_str = tmp_buf.getvalue()
                 console_message = ConsoleMessage(
                     worker_id=self.worker_id,
-                    timestamp=datetime.datetime.now(),
+                    timestamp=datetime.datetime.now(datetime.timezone.utc),
                     msg_type=ConsoleMessageType.PRINT,
```

Comment:
We still need to add a test case to ensure after the fix, the console message with timestamp does not have the serialization problem with the updated betterproto package.